web/app.js

```diff
@@ -1114,6 +1115,9 @@ var PDFViewerApplication = {
       this.page = 1;
     }
 
+    this.toolbar.setPageNumber(this.pdfViewer.currentPageNumber,
+                               this.pdfViewer.currentPageLabel);
```

Comment:
I think that this should probably be moved to after line https://github.com/mozilla/pdf.js/pull/7789/files#diff-529d1853ee1bba753a0fcb40ea778723R868 instead, which may also fix https://github.com/mozilla/pdf.js/pull/7789#issuecomment-258886196.